pkg/scheduler/core/extender.go

```diff
@@ -298,7 +249,7 @@ func (h *HTTPExtender) convertPodUIDToPod(
 
 // convertToNodeNameToMetaVictims converts from struct type to meta types.
 func convertToNodeNameToMetaVictims(
-	nodeToVictims map[*v1.Node]*extenderv1.Victims,
+	nodeToVictims map[string]*extenderv1.Victims,
 ) map[string]*extenderv1.MetaVictims {
 	nodeNameToVictims := map[string]*extenderv1.MetaVictims{}
```

Comment:
It might be better understood if we rename nodeNameToVictims to nodeToMetaVictims here.

Reply:
Done.